### DISCUSSION TITLE
Flix Lang Kit?

### DISCUSSION THREAD

**Discussion by 3xau1o** [sdkman/sdkman-cli#1162] [Ideas]
What would take to list Flix compiler kit in SDKMan?

https://flix.dev/

**Comment by marc0der** [comment]
So there are two possibilities here. The first is to run this as a community-driven candidate through our database migrations project. This often results in the latest version not being available immediately until someone raises a PR to add it.The second (and better) option is to engage with the maintainers of Flix and to ask them to start using our vendor API to publish their own versions.

**Reply by 3xau1o** [reply]
@magnus-madsen would you be interested?

**Reply by magnus-madsen** [reply]
There was some effort here: https://github.com/flix/flix/issues/4561 but not sure what the status is.
